Shop-Facade-Open-Api-Spec.yaml

```
@@ -0,0 +1,1108 @@
+                type: string
+                format: RFC1123
+          content: {}
+        "304":
+          description: Not modified - Catalogue hasn't changed since the If-Modified-Since
+            date
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content: {}
+        "401":
+          description: Token Authentication Failed
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content: {}
+        "403":
+          description: "Forbidden or Quota Exceeded.<br>This will either be because:\
+            \ <br> - you have no permission to use this API (in which case the body\
+            \ will be empty) <br> - the quota has been exceeded"
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+        "404":
+          description: Catalogue ID not found
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+            application/json:
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: catalogueId
+                  description: Catalogue ID not found
+        "429":
+          description: Too Many Requests.
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ApiManagementError'
+            application/json:
+              example:
+                statusCode: 429
+                message: Rate limit is exceeded. Try again in 1 seconds.
+        "500":
+          description: Internal Server Error.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/exceptionDescription'
+                
+components:
+  schemas:
+    exceptionDescription:
+      type: object
+      properties:
+        correlationId:
+          type: string
+    errorDescription:
+      type: object
+      properties:
+        correlationId:
+          type: string
+        errors:
+          type: array
+          items:
+            $ref: '#/components/schemas/fieldError'
+    ApiManagementError:
```

lowercase a on apiManagementError to keep it consistent with the others.

```suggestion
    apiManagementError:
```

---

update it and its references.